packages/blocks/src/page-block/edgeless/frame-manager.ts

```diff
@@ -23,6 +23,16 @@ import { isFrameBlock } from './utils/query.js';
 const MIN_FRAME_WIDTH = 800;
 const MIN_FRAME_HEIGHT = 640;
 const FRAME_PADDING = 40;
+
+export const removeContainedFrames = (frames: FrameBlockModel[]) => {
```

Comment:
Please use `export function` by default.

Reply:
Get it!